solo-project/client/src/components/app/add-event/component.js

```diff
@@ -0,0 +1,128 @@
+/*global google*/
+import React from 'react';
+import PropTypes from 'prop-types';
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
+import 'react-datepicker/dist/react-datepicker.css';
+import Geosuggest from 'react-geosuggest';
+
+import './index.css';
+
+export class component extends React.Component {
+  static propTypes = {
+    getEvents: PropTypes.func,
+    createEvent: PropTypes.func
+  };
+
+  state = {
+    date: moment(),
+    title: '',
+    description: '',
+    location: '',
+    coords: { lat: null, lng: null },
+    ageFrom: '',
+    ageTo: '',
+    price: '',
+    error: null
+  };
+
+  handleChange = date => {
+    this.setState({ date: date });
+  };
+
+  handleSuggestSelect = place => {
+    const { lat, lng } = place.location;
+    this.setState({
+      location: place.gmaps.formatted_address,
+      coords: [lat, lng]
+    });
+  };
+
+  handleChangeInput = event => {
+    event.preventDefault();
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleSubmitForm = event => {
+    event.preventDefault();
+    this.props
+      .createEvent({
```

Comment:
Here you are passing an object will all your states expect for error. A cool way to do the same thing with less code could be: 
```
handleSubmitForm = event => {
    event.preventDefault();
    const { error, ...eventDetail } = this.state;
    this.props
      .createEvent(eventDetail)
```

Check what the rest operator does!